### PR TITLE
Some comments on the mutation appendix

### PR DESCRIPTION
**Pull request by gregorgorjanc**
I have provided comments as inline LaTex comments. I think this is mostly for @petrelharp

### CODE REVIEW

**Comment by jeromekelleher** [comment]
Thanks @gregorjanc - over to you @petrelharp?

(We can do this via overleaf if that would be easier?)

**Comment by petrelharp** [comment]
Thanks, @gregorgorjanc! I can revise (and no need for overleaf).

**Comment by jeromekelleher** [comment]
OK, let's merge this so and you can delete the comments as you're revising @petrelharp ?

File: paper.tex [hunk 1]
```diff
@@ -1784,6 +1792,7 @@ \section*{Mutation generation}
 process with rate $\beta$.
 All finite-state models (implemented under the generic \texttt{MatrixMutationModel} class)
 work in this way: mutations are placed at the maximum mutation rate,
+% I think I understand what you want to say here, but it would be clearer if an example would have been given - similar to the next section.
```
Comment (petrelharp) [inline paper.tex:1795]: But we just gave an example, @gregorgorjanc - with X and Y. Would it be clearer if the example was with nucleotides?

Comment (gregorgorjanc) [reply paper.tex:1795]: Yes! I couldn’t follow your argument with thinning Poisson, but that doesn’t matter really, I liked the section below more. Feel free to ignore my ranting;)

Comment (petrelharp) [reply paper.tex:1795]: Well, I've changed the example to nucleotides - see if you think it's more clear?

**Pull request merged**